runners/spark/src/main/java/org/apache/beam/runners/spark/stateful/SparkGroupAlsoByWindowViaWindowSet.java

```diff
@@ -294,8 +295,6 @@
                               reduceFn,
                               options.get());
 
-                      outputHolder.clear(); // clear before potential use.
```

Comment:
Looks like removing this invocation means `org.apache.beam.runners.spark.stateful.SparkGroupAlsoByWindowViaWindowSet.OutputWindowedValueHolder#clear` is no longer used, can we remove this method?

Reply:
removed